File: src/BIDIM.Common/Models/IndividualViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BIDIM.Common.Models
{
    public class IndividualViewModel
    {
        public Guid Id { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public DateTime DoB { get; set; }

        public int Age { get; set; }

        public string Gender { get; set; }

        public string ContactNumber { get; set; }

        public int HouseholdId { get; set; }

        public bool IsActive { get; set; }

        public bool IsDeceasedByDisease { get; set; }
    }
}

File: src/BIDIM.WebApp/src/app/_viewModels/IndividualViewModel.ts
export interface IIndividualList {
  Id: string;
  FullName: string;
  Gender: string;
  HouseholdFamilyName: string;
  IsActive: string;
}

export interface IIndividualViewModel {
  Id: string;
  FirstName: string;
  LastName: string;
  DoB: string;
  Age: number;
  Gender: string;
  ContactNumber: string;
  HouseholdId: number;
  IsActive: boolean;
  IsDeceasedByDisease: boolean;
}

export interface IIndividualDropdownList {
  Id: string;
  FullNameHousehold: string;
}
